docker-compose/docker-compose.yaml

```diff
@@ -18,7 +18,7 @@ services:
   caddy:
     container_name: caddy
     image: 'index.docker.io/caddy/caddy:alpine@sha256:83d9aa7a5f1bbcc0fc1b4720c183a5ec53dae7dc5d9fa555daf3db345010e7f9'
-    cpus: 4
```

Comment:
What's the impact of using cpu_shares on our Grafana metrics / monitoring?

I'd be interested in trying this out on a managed instance first, then adopting it as the standard.

Reply:
Shouldn't have any impact. Our cAdvisor metrics look at CPU usage. They might compare it against cpu_limit though which could be an issue

Reply:
https://github.com/google/cadvisor/blob/master/docs/storage/prometheus.md & ctrl-f limits. 
There is no `cpu_limits` only CFS periods so I highly doubt any cAdvisor issues would be affected by this